addons/oceanbase/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: oceanbase
+name: ob-ce
```

Comment:
change the addon oceanbase-cluster dir name too 
and update kubeblcoks addon name